TestCoreApp_Elliott/Program.cs

```diff
@@ -26,16 +30,35 @@
 
 app.UseAuthorization();
 
-app.MapAreaControllerRoute(
+// MUST BE CALLED before UseEndpoints
+app.UseSession();
+
+// This will catch:
+//	https://localhost:7044/Assignment6_1/Assignment6_1/Assignment6_1/
+//	https://localhost:7044/Assignment6_1/Assignment6_1/Assignment6_1/9
+app.MapControllerRoute(
+	name: "assignment61",
+	pattern: "{area:exists}/{controller=Assignment6_1}/{action=Assignment6_1}/{accessLevel?}");
+
+// This will catch:
+//	https://localhost:7044/olympics/home/index/cat/all/game/all
+app.MapControllerRoute(
 	name: "olympics",
-    areaName: "Olympics",
-	pattern: "Home/{action=Index}/cat/{activeCat}/game/{activeGame}");
+	pattern: "{area:exists}/{controller=Home}/{action=Index}/cat/{activeCat}/game/{activeGame}");
 
-app.MapAreaControllerRoute(
-    name: "assignment",
-    areaName: "Assignment6_1",
-    pattern: "Assignment6_1/{action=Assignment6_1}/{accessLevel?}");
+// This will catch:
+//	https://localhost:7044/Olympics
+//	https://localhost:7044/Olympics/Home
+//	https://localhost:7044/Olympics/Home/Index
+app.MapControllerRoute(
+	name: "olympicsDefault",
+	pattern: "{area:exists}/{controller=Home}/{action=Index}");
 
+// This will catch:
+//	https://localhost:7044/
+//	https://localhost:7044/Home
+//	https://localhost:7044/Home/Index
+//	https://localhost:7044/Home/Index/1
 app.MapControllerRoute(
     name: "default",
     pattern: "{controller=Home}/{action=Index}/{id?}");
```

Comment:
All of these are for routing. It looks like they are using `{area:exists}` instead of the MapArea method. I am not certain I like the first option but they will both do the same.